media-collections/create-rename-delete-search-and-download-as-zip.md

```diff
@@ -26,7 +26,7 @@ All media collections whose names either match exactly or begin with the provide
 
 ## Download media collections as zip
 
-ImageKit allows you to download multiple media collections and all of their contents, including files, folders, and sub-folders, as a single zip file. The downloaded zip maintains the original folder structure.
+ImageKit allows you to download multiple media collections and all of their contents, including files, folders, and sub-folders, as a single zip file. The downloaded zip file preserves the folder hierarchy structure of the media collections.
```

Comment:
`hierarchy structure` sounds redundant, just `folder hierarchy` or `folder structure` should be sufficient.

Reply:
done